main.py

```diff
@@ -51,6 +58,11 @@ async def request_classification(request: Request):
     image_id = form.image_id
     model_id = form.model_id
     classification_scores = classify_image(model_id=model_id, img_id=image_id)
+
+    file_path = "app/static/json_results.json"
+    with open(file_path, "w") as file:
+        json.dump(classification_scores, file)
```

Comment:
What happens if multiple users ask for a classification job? The file would be overwritten. Isn't it best to return the results directly? Or, if you reuse them later for other things, another way to do it would be to store the results with a unique name and clean up after the response is returned to the user (or after a while anyway).